MASTER FUNCTIONS LIBRARY.vbs

```diff
@@ -5885,7 +5885,18 @@ function determine_130_percent_of_FPG(footer_month, footer_year, hh_size, fpg_13
 
 	If IsNumeric(hh_size) = True Then							'error handling to ensure that HH size is a number
 		hh_size = hh_size*1
-		If DateDiff("d", #10/1/2022#, month_to_review) >= 0 Then				'these are the associated amounts
+		If DateDiff("d", #10/1/2023#, month_to_review) >= 0 Then				'these are the associated amounts
+			If hh_size = 1 Then fpg_130_percent = 1580
+			If hh_size = 2 Then fpg_130_percent = 2137
+			If hh_size = 3 Then fpg_130_percent = 2694
+			If hh_size = 4 Then fpg_130_percent = 3250
+			If hh_size = 5 Then fpg_130_percent = 3807
+			If hh_size = 6 Then fpg_130_percent = 4364
+			If hh_size = 7 Then fpg_130_percent = 4921
+			If hh_size = 8 Then fpg_130_percent = 5478
+
+			If hh_size > 8 Then fpg_130_percent = 5052 + (557 * (hh_size-8))
```

Comment:
Typo - should be 5478 + (557 * (hh_size-8)) to reflect 2023 amount.

Reply:
Good catch @MDR22, fixed!